onnxruntime/core/providers/cpu/quantization/quantize_linear.cc

```diff
@@ -79,16 +79,28 @@ static void PrepareForQDQ(const TensorShape& input_shape,
 
 #define REGISTER_DEQUANTIZELINEAR(T)                                         \
   ONNX_CPU_OPERATOR_TYPED_KERNEL(                                            \
+      DequantizeLinear,                                                      \
+      21,                                                                    \
+      T,                                                                     \
+      KernelDefBuilder()                                                     \
+          .TypeConstraint("T1", DataTypeImpl::GetTensorType<T>())            \
+          .TypeConstraint("T2", {DataTypeImpl::GetTensorType<float>(),       \
+                                 DataTypeImpl::GetTensorType<MLFloat16>()}), \
+      DequantizeLinear<T>);
+
+#define REGISTER_DEQUANTIZELINEAR_VERSIONED(T)                               \
```

Comment:
nit: I wonder if ` REGISTER_DEQUANTIZELINEAR_VERSION_20` would be a better name for this? Just trying to understand the code here.

Reply:
Yeah, the name is confusing. What do think about `REGISTER_DEQUANTIZE_LINEAR_VERSION_19_20` for this macro since that's range?

Reply:
Sure, that sounds good (my fault for suggesting just 20!)

Reply:
We discussed and agreed to handle the rename separately. I'll do it the follow-up int4 PR.